### PR TITLE
Improve error message

### LINKED ISSUE #4102: Mac: dotnet dev-certs https --trust does not explain how to undo this
When you run this command, it does not say what actually is taking place, it leaves a critical element, which is needed later to undo this:

```
$ dotnet dev-certs https --trust
Trusting the HTTPS development certificate was requested. If the certificate is not already trusted we will run the following command:
'sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain <<certificate>>'
This command might prompt you for your password to install the certificate on the system keychain.
```

I could not figure out what certificate was added to my system, it took me a while to debug this as the actual certificate is replaced with the string `<<certificate>>` above.

### PR DESCRIPTION
**Pull request by HaoK**
Fixes https://github.com/dotnet/aspnetcore/issues/4102

### CODE REVIEW

**Review by Tratcher — APPROVED** [review]

File: src/Tools/dotnet-dev-certs/src/Program.cs [hunk 1]
```diff
@@ -363,9 +363,10 @@ private static int EnsureHttpsCertificate(CommandOption exportPath, CommandOptio
             {
                 if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
                 {
+                    var certPath = exportPath.HasValue() ? exportPath.Value() : "<<certificate>>";
                     reporter.Warn("Trusting the HTTPS development certificate was requested. If the certificate is not " +
                         "already trusted we will run the following command:" + Environment.NewLine +
-                        "'sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain <<certificate>>'" +
+                        $"'sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain {certPath}'" +
```
Comment (javiercn) [inline src/Tools/dotnet-dev-certs/src/Program.cs:369]: How does this fix the issue?

I would suggest that the message instead becomes something like: "To undo these changes ...." and suggest either running `dotnet dev-certs https --clean` to remove the cert (and the trust) or to display the right command on mac os to remove the certificate from the list of trusted system certificates.

Comment (HaoK) [reply src/Tools/dotnet-dev-certs/src/Program.cs:369]: So just add a sentence on the end that says "To undo these changes, run dotnet dev-certs https --clean" and leave it as `<<cert>>` always?

Comment (HaoK) [reply src/Tools/dotnet-dev-certs/src/Program.cs]: Okay updated it to add: `To undo these changes: 'sudo security remove-trusted-cert -d <<certificate>>'`
